packages/sdk/src/Actions/EventAction.test.ts

```diff
@@ -119,7 +119,7 @@ describe('EventAction', () => {
         filterType: FilterType.EQUAL,
         fieldType: PrimitiveType.ADDRESS,
         fieldIndex: 2,
-        filterData: accounts.at(1)!.account.toUpperCase(),
+        filterData: accounts[1].account.toUpperCase(),
```

Comment:
I don't think this refactor solves the core issue which is that the item is potentially undefined. Is the bracket access a preference thing?

Reply:
It does because the accounts are fixed-length tuple now

Reply:
https://github.com/rabbitholegg/boost-protocol/pull/101/files#diff-7706002d7a8917202680898cc3c6ba463b21385a434f88d97c3939fd61bc2ecfR10-R21